src/library/librarycontrol.h

```diff
@@ -65,27 +79,51 @@ class LibraryControl : public QObject {
     void slotDecrementFontSize(double v);
 
   private:
-    Library* m_pLibrary;
-
-    ControlObject* m_pSelectNextTrack;
-    ControlObject* m_pSelectPrevTrack;
-    ControlObject* m_pSelectTrack;
-
-    ControlObject* m_pSelectSidebarItem;
-    ControlObject* m_pSelectPrevSidebarItem;
-    ControlObject* m_pSelectNextSidebarItem;
-
-    ControlObject* m_pToggleSidebarItem;
-    ControlObject* m_pLoadSelectedIntoFirstStopped;
-    ControlObject* m_pAutoDjAddTop;
-    ControlObject* m_pAutoDjAddBottom;
-
-    ControlObject* m_pFontSizeKnob;
-    ControlPushButton* m_pFontSizeIncrement;
-    ControlPushButton* m_pFontSizeDecrement;
-
-    WLibrary* m_pLibraryWidget;
-    WLibrarySidebar* m_pSidebarWidget;
+    Library* m_pLibrary = nullptr;
+
+    // Simulate pressing a key on the keyboard
+    void emitKeyEvent(QKeyEvent&& event);
+
+    // Controls to navigate vertically within currently focussed widget (up/down buttons)
+    ControlPushButton* m_pMoveUp = nullptr;
```

Comment:
At my work I use the convention that only the member variables which depend on the input arguments go in the initializer list. If the variable should be initialized with a value which is known at compile time then we always put the value in at the point of declaration in the header file.

This has two advantages:
1) It requires less code

2) It is less maintenance work because the order of declaration in the header file in principle should be the same as the order of initialization.

http://stackoverflow.com/questions/1242830/constructor-initialization-list-evaluation-order

Because of this stupid c++ rule it's a good idea to only put the necessary members into the initializer list for your own sanity